src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -518,6 +518,13 @@ Namespace Microsoft.CodeAnalysis.Operations
             End If
 
             Dim conversion As Conversion = CreateConversion(boundTryCast)
+
+            If conversionInformation.ConversionKind <> boundTryCast.ConversionKind Then
+                ' If the conversionInformation has a different conversion kind, we had a nested conversion of an expression tree
+                ' and we need to use the nested information for our final conversion
+                conversion = New Conversion(KeyValuePair.Create(conversionInformation.ConversionKind, conversion.Method))
```

Comment:
>conversion.Method [](start = 102, length = 17)

Why is this the right thing to combine method and kind from different sources? Does ```conversionInformation``` include method? Could we include it there and simply always rely on information in ```conversionInformation```?

Reply:
Yeah, we can do that.

Reply:
@AlekseyTs updated.